.github/workflows/dissect-ci-template-self-hosted.yml

```
@@ -15,12 +15,22 @@ jobs:
   build:
     runs-on: ubuntu-latest-sh
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          lfs: true
-      - run: git lfs pull
-      - uses: actions/setup-python@v4
+
+      # CACHING LFS OBJECTS
+      - name: Create LFS file list
+        run: git lfs ls-files --long | cut -d ' ' -f1 | sort > .lfs-assets-id
+      - name: Cache LFS
+        uses: actions/cache@v4
+        with:
+          path: .git/lfs
+          key: lfs-${{ hashFiles('.lfs-assets-id') }}-v1
+      - name: Git LFS pull
```

```suggestion
      - name: Pull LFS
```
To stay with the naming of the others (`<Action> LFS`)